nmdc_runtime/site/export/ncbi_xml.py

```diff
@@ -366,7 +366,14 @@ def set_fastq(
                             )
                             # Currently, we are making the assumption that only one instrument
                             # is used to sequence a Biosample
-                            instrument_id = ntseq.get("instrument_used", "")[0]
+                            instrument_used: Union[str, List[str]] = ntseq.get(
+                                "instrument_used", []
+                            )
+                            if not instrument_used:
+                                instrument_id = None
+                            else:
+                                instrument_id = instrument_used[0]
```

Comment:
Now that the fallback value is an empty list (instead of an empty string), I think (without knowing where this object comes from) the value will be a list of strings in all situations.

```suggestion
                            instrument_used: List[str] = ntseq.get(
                                "instrument_used", []
                            )
                            if not instrument_used:
                                instrument_id = None
                            else:
                                instrument_id = instrument_used[0]
```